docs/docs/installation_guide/installation/ami-install.md

```diff
@@ -23,4 +23,8 @@ In the Amazon EC2 service console, select AMI in the left-hand navigation. You s
 
 <img src={useBaseUrl('img/deployment/installation/AMI.png')} />
 
+:::note
+Because Hongkong region do not support t2 instance type, please follow this [guide](https://github.com/awslabs/service-workbench-on-aws-cn/discussions/76) to generate Windows AMI in Hongkong region.
```

Comment:
It's a workaround to build Windows AMI in HK region, what's the root cause of failing building AMI in T3 instance?

Reply:
Yes, it is a workaround for Hongkong Region, the root cause of failing building AMI in T3 instance need more time to research. Submitted an [issue](https://github.com/awslabs/service-workbench-on-aws-cn/issues/79) to track this issue. Once the issue is fixed, will modify the doc.